vmdb/app/models/vm_or_template.rb

```diff
@@ -416,6 +416,9 @@ def self.invoke_tasks_local(options)
     elsif ["create_snapshot"].include?(options[:task])
       options[:invoke_by] = :task
       args = [options[:name], options[:description], options[:memory]]
+    elsif ["retire_now"].include?(options[:task])
```

Comment:
This can all be done with a simple case statement instead of these includes...

```ruby
case options[:task]
when "scan", "sync" then ...
when "remove_snapshot", "revert_to_snapshot" then ...
when "create_snapshot" then ...
when "retire_now" then ...
end

```